doc/guide/cockpit-guide.xml

```diff
@@ -25,6 +25,7 @@
     <xi:include href="https.xml"/>
     <xi:include href="listen.xml"/>
     <xi:include href="sso.xml"/>
+    <xi:include href="authentication.xml"/>
```

Comment:
Lets put authentication.xml content before sso.xml. Hopefully we can make this coherent soon.